zipline-kotlin-plugin/src/main/kotlin/app/cash/zipline/kotlin/ir.kt

```diff
@@ -102,6 +104,16 @@ internal val IrSimpleFunction.signature: String
     }
   }
 
+/** See signatureHash(). */
+internal val IrSimpleFunction.id: String
+  get() {
+    // This would be more compact with Okio, but adding that dependency is tricky.
+    val signatureUtf8 = signature.encodeToByteArray()
+    val sha256 = MessageDigest.getInstance("SHA-256").digest(signatureUtf8)
+    val sha256Prefix = sha256.sliceArray(0 until 6)
+    return String(Base64.getEncoder().encode(sha256Prefix))
```

Comment:
I started with Okio but the tools didn’t like that. Easier to just do this the long way.
```
 java.lang.NoClassDefFoundError: okio/ByteString
	at app.cash.zipline.kotlin.IrKt.getId(ir.kt:108)
	at app.cash.zipline.kotlin.AdapterGenerator$irZiplineFunctionClass$2$4$1.invoke(AdapterGenerator.kt:528)
	at app.cash.zipline.kotlin.AdapterGenerator$irZiplineFunctionClass$2$4$1.invoke(AdapterGenerator.kt:521)
	at app.cash.zipline.kotlin.IrKt.irDelegatingConstructorCall(ir.kt:217)
	at app.cash.zipline.kotlin.AdapterGenerator$irZiplineFunctionClass$2$4.invoke(AdapterGenerator.kt:521)
	at app.cash.zipline.kotlin.AdapterGenerator$irZiplineFunctionClass$2$4.invoke(AdapterGenerator.kt:520)
	at app.cash.zipline.kotlin.IrKt$irConstructorBody$1.invoke(ir.kt:198)
	at app.cash.zipline.kotlin.IrKt$irConstructorBody$1.invoke(ir.kt:194)
```